ansible/roles/insight/tasks/main.yml

```diff
@@ -20,10 +20,11 @@
     restarted: yes
     pull: yes
 
-- name: check Insight container
-  assert:
-    that: 'insight.insight.state.running'
+# Disable broken checks
+# - name: check Insight container
```

Comment:
Because it doesn't work, it always interrupts execution with `fatal: [web-1]: FAILED! => {"msg": "The conditional check 'insight.insight.state.running' failed. The error was: error while evaluating conditional (insight.insight.state.running): 'insight' is undefined"}`

I tried debugging with 
```
- name: debug
  ansible.builtin.debug:
    var: 'insight'
```
But the variable does not exist.
```
ok: [web-1] => {
    "insight": "VARIABLE IS NOT DEFINED!"
}
```
The entire ansible documentation for this function is `This module asserts that given expressions are true with an optional custom message.` but this is meaningless to me, obviously it did not set it true because the variable is undefined. I can't see how this was supposed to work or what it was supposed to do.

Reply:
It seems Insight's containers are not running